TrackerEvents.cs

```diff
@@ -91,32 +68,22 @@ internal static void RegisterRunEventHandlers(DynamoModel model)
         /// <param name="e"></param>
         private static void OnEvaulationCompleted(object sender, EvaluationCompletedEventArgs e)
         {
-            /*
-            if (FileName == null)
-            {
-                FileName = "New Graph";
-            }
-            */
-
-            // @todo if EvaluationCount greater than 1..
+            // Display a MessageBox to the user
+            // @todo Remove this as it's for testing purposes only
+            string filename = ExportData.filename;
+            MessageBox.Show($"The current Graph name is {filename}");
 
-            // WORKS
-            MessageBox.Show($"The current Graph name is {FileName}");
-
-            var dataToExport = ExportData.Export(FileName, DynamoVersion);
-            ExportSheets.Execute(dataToExport);            
+            // When graph evaluation is complete
+            // Get all the data we want to store and then store it
+            var dataToExport = ExportData.Export();
+            ExportSheets.Execute(dataToExport);
```

Comment:
Both these feel very un-C# like, would suggest something that is more OO and not use so many `static` methods.